NICE.Identity.Management/ClientApp/src/views/Organisation/Organisation.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from "react";
+import { RouteComponentProps, Link } from "react-router-dom";
+import { StaticContext } from "react-router";
+
+import { OrganisationType, UserType } from "../../models/types";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { PageHeader } from "@nice-digital/nds-page-header";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { Alert } from "@nice-digital/nds-alert";
+
+import { fetchData } from "../../helpers/fetchData";
+import { isDataError } from "../../helpers/isDataError";
+import { Endpoints } from "../../data/endpoints";
+
+import { Button } from "@nice-digital/nds-button";
+
+import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
+import styles from "./Organisation.module.scss";
+
+import { UserStatus } from "../../components/UserStatus/UserStatus";
+import { ToFormattedDateString } from "../../helpers/dateHelpers";
+
+type TParams = { id: string };
+
+type LocationState = {
+	hasBeenEdited: boolean;
+};
+
+type OrganisationProps = Record<string, unknown> &
+	RouteComponentProps<TParams, StaticContext, LocationState>;
+
+type OrganisationState = {
+	organisation: OrganisationType;
+	users: Array<UserType>;
+	error?: Error;
+	redirect: boolean;
+	isLoading: boolean;
+};
+
+export class Organisation extends Component<OrganisationProps, OrganisationState> {
+		constructor(props: OrganisationProps) {
+			super(props);
+	
+			this.state = {
+				organisation: {} as OrganisationType,
+				users: [],
+				redirect: false,
+				isLoading: true,
+			};
+		}
```

Comment:
Could we add a document title in? Think this is missing off other components, so apologies if it was missing in the one you used as reference.

```suggestion
		constructor(props: OrganisationProps) {
			super(props);
	
			this.state = {
				organisation: {} as OrganisationType,
				users: [],
				redirect: false,
				isLoading: true,
			};
			
			document.title = "NICE Accounts - Organisation details";
		}
```